blog/2023-07-23-openhab-4-0-release.md

```diff
@@ -0,0 +1,199 @@
+
+Imagine you want to know which of your rules and pages use a specific Item. Just type the Item name in the developer sidebar search, and you’ll find all occurrences of the search query in pages, scenes, scripts and rules. 
+Even Blockly and the code of script actions are now searchable.
+
+<!-- Add image here -->
+
+For an exact list of entities the search checks your query against, please have a look at [the documentation](https://openhab.org/docs/tutorial/tips-and-tricks.html#search).
+
+### Open openHAB to thousands of free new icons
+
+_Laurent Garnier, openHAB Maintainer_
+
+#### New icon sources available
+
+In addition to the icons already provided via the openHAB server (classic icon set + custom user icons), you have now easy access for your items and sitemap elements to the [Material icons](https://fonts.google.com/icons?icon.set=Material+Icons), [Framework7 icons](https://framework7.io/icons/) and [over 150,000 icons provided by Iconify](https://icon-sets.iconify.design/).
+
+Main UI supports all these icon sources while Basic UI currently supports all of them except the Framework7.
+Note that the Iconify option requires internet connectivity on the client to access the external API.
+The web browser will cache the retrieved icons to limit the requests and speed up the rendering.
+Basic UI provides a setting to enable this option which is disabled by default.
+
+Dynamic icons based on the item state remains a feature only available with the openHAB icon sets.
+
+For more details including the new category/icon syntax, please read [the documentation](https://www.openhab.org/docs/configuration/items.html#icons).
+
+#### Managing your icon color
+
+You can now control the icon color associated with any user interface element of a sitemap using the "iconcolor" parameter. The syntax and capabilities behind this parameter are the same as for the "labelcolor" and "valuecolor" parameters, which means you can define multiple conditions leading to different colors.
+
+This is not possible with icons from the classic icon set because these icons have a hard-coded color palette, but it is possible with icons from other icon sources and with added user SVG icons containing "currentColor" as the fill color.
+Basic UI supports this color feature for openHAB icons only if an option is enabled.
+Color support for other icon sources is supported by default.
+
+For more details, please read [the documentation](https://www.openhab.org/docs/ui/sitemaps.html#label-value-and-icon-colors).
+
+#### Icons provided by bindings
+
+A binding can now make specific icons related to channels available to the user, especially when no appropriate category/icon is available in the classic icon set.
+Few bindings have already integrated this feature, for example the [OpenUV binding](https://www.openhab.org/addons/bindings/openuv/).
+
+## Addon Enhancements
+
+_Jacob Laursen, openHAB Maintainer_
+
+18 new bindings are included, supporting new devices and services:
```

Comment:
```suggestion
17 new addons are included, supporting new devices and services:
```

to be precise. 14 bindings, 2 transformations, 1 persistence and 1 automation - the automation is jsscriptingnashorn, so we probably shouldn't count it.